Compilation warning about unused variable

```
In file included from vroom_fwf.cc:7:
./columns.h:125:10: warning: variable 'to_parse' set but not used [-Wunused-but-set-variable]
  size_t to_parse = 0;
```

https://github.com/tidyverse/vroom/blob/89b6aac8c9c07981a52c28251578deca11810ebd/src/columns.h#L125C10-L131

Looks real? i.e. `to_parse` doesn't actually get used beyond this loop

--- a/src/columns.h
+++ b/src/columns.h
@@ -92,7 +92,6 @@ inline cpp11::list create_columns(
     size_t num_threads) {
 
   R_xlen_t num_cols = idx->num_columns();
-  auto num_rows = idx->num_rows();
 
   auto locale_info = std::make_shared<LocaleInfo>(locale);
 
@@ -121,14 +120,6 @@ inline cpp11::list create_columns(
       locale_info,
       guess_max,
       altrep);
-
-  size_t to_parse = 0;
-  for (R_xlen_t col = 0; col < num_cols; ++col) {
-    auto collector = my_collectors[col];
-    if (collector.use_altrep()) {
-      to_parse += num_rows;
-    }
-  }
 
   for (R_xlen_t col = 0; col < num_cols; ++col) {
     auto collector = my_collectors[col];

